.ibm/pipelines/openshift-ci-tests.sh

```diff
@@ -343,6 +354,30 @@ install_tekton_pipelines() {
   fi
 }
 
+install_rhdh_operator() {
+  local dir=$1
+  local namespace=$2
+
+  configure_namespace $namespace
+  
+  # Make sure script is up to date
+  rm -f /tmp/install-rhdh-catalog-source.sh
+  curl -L https://raw.githubusercontent.com/rm3l/redhat-developer-hub-operator/refs/heads/RHIDP-4934--unable-to-use-ci-install-script-in-prow-containers-where-podman-is-restricted/.rhdh/scripts/install-rhdh-catalog-source.sh > /tmp/install-rhdh-catalog-source.sh
```

Comment:
```suggestion
  curl -L https://raw.githubusercontent.com/redhat-developer/rhdh-operator/refs/heads/main/.rhdh/scripts/install-rhdh-catalog-source.sh > /tmp/install-rhdh-catalog-source.sh
```

I've just merged https://github.com/redhat-developer/rhdh-operator/pull/404